mlflow/tracing/fluent.py

```diff
@@ -260,7 +301,15 @@ def pagination_wrapper_func(number_to_get, next_page_token):
 
     get_display_handler().display_traces(results)
 
-    return results
+    traces_df = traces_to_df(results)
```

Comment:
Curious how much overhead this traces->df conversion adds when there are many traces (e.g. 1000).

Reply:
+1, wondering if we can make dataframe conversion opt-in or a separate API